packages/math-input/src/keypad/index.jsx

```diff
@@ -191,12 +191,12 @@ const styles = theme => ({
     }
   },
   comma: {
-    fontSize: '2.2rem',
```

Comment:
These do need to be bigger. also `labelButton`. `inherit` won't do the trick. We could set pixel size directly instead?

Reply:
You are right.
I would avoid setting pixel size, I'll use percentage instead.